src/frontends/pytorch/src/op_table.cpp

```diff
@@ -364,8 +364,8 @@ const std::map<std::string, CreatorFunction> get_supported_ops_ts() {
         // aten::chunk - Supported in limited set of patterns
         {"aten::clamp", op::translate_clamp},
         {"aten::clamp_", op::inplace_op<op::translate_clamp>},
-        {"aten::clamp_max", op::translate_1to1_match_2_inputs_align_types<opset10::Minimum>},
-        {"aten::clamp_min", op::translate_1to1_match_2_inputs_align_types<opset10::Maximum>},
+        {"aten::clamp_max", op::translate_1to1_match_2_inputs_align_to_lhs<opset10::Minimum>},
```

Comment:
Do we have tests for this case when we need to align to lhs rather then common type?

Reply:
Thanks for noticing it, after some tests I think this LHS can be replaced completely by using regular align to common type after some tests, my bad that I didn't catch it earlier.
This conversion was added in early development of this PR  due to weird precision loses in one of precommit HF model that I tracked back to clamp/comparison ops, however it seem that real issue was due to other problems found later in development and this fix probably only masked this issue to be found later.